process/project-lifecycle.md

```diff
@@ -0,0 +1,161 @@
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage Projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project
+* Facilitate alignment with existing Working Groups or Projects if (and only if) this is desired
+* Nurture Projects (e.g. via OpenSSF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct, and IP Policy requirements.
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Sandbox Entry Requirements
+
+Projects being submitted to the OpenSSF at the sandbox level are intended to be the entry point for early stage projects and are not required to undergo due diligence.
+
+Sandbox Projects should be early-stage projects that the OpenSSF TAC or parent Working Group believes warrant experimentation.
+
+* New Projects that are designed to extend one or more OpenSSF Projects with functionality or interoperability libraries.
+* Independent projects that fit the OpenSSF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+* Projects commissioned or sanctioned by the OpenSSF, including initial code for OpenSSF WG collaborations, and "experimental" projects
+* Any project that realistically intends to join OpenSSF Incubation in future and wishes to lay the foundations for that
+
+#### Project Graduation Process: Sandbox to Incubating
+
+Incubating Projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC or parent WG sponsor. Once the diligence is complete and the proposal made, two weeks are allocated for public comment before a TAC vote is called.
+
+#### Submission Forms for Sandbox
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+#### Submission Forms for Sandbox to Incubating
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+### Incubating
+
+![Incubating](/process/incubation-process.png)
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Incubating Entry Requirements
+
+To be accepted to incubating stage, a project must meet the sandbox stage requirements plus:
+
+* Document that it is being used successfully in production by at least three independent end users which, in the TAC’s judgement, are of adequate quality and scope.
```

Comment:
is the intent to only permit end users as a qualifier?  Depending on the nature of a given project in fixing upstream security issues, i can't imagine we're terribly concerned about end users, rather we'd like to focus on adopters which may include other open source security projects who inherently are not end users. (This may be reuse from other foundations where End Users are official things within the construct of the foundation).

Reply:
Makes sense to me. Changing to "adopters. Thanks.